package.json

```diff
@@ -24,6 +24,7 @@
   "dependencies": {
     "jquery": "^3.3.1",
     "node-hid": "^0.7.2",
+    "os": "^0.1.1",
```

Comment:
`os` is a native nodejs module (https://nodejs.org/dist/latest-v10.x/docs/api/os.html), please remove as dependency